recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
     git_rev: v{{ version }}
 
 build:
-  number: 3
-  skip: true  # [win or cuda_compiler_version in (undefined, "None")]
+  number: 5
+  skip: true  # [win or not linux64 or cuda_compiler_version in (undefined, "None", "11.2")]
```

Comment:
CuPy v8.4.0 officially supports up to CUDA 11.1. Let's build for CUDA 11.2 when the bot sends an update PR for v8.5.0.

Reply:
Makes sense

Do we know why the build number went from 3 to 5?

Reply:
...apparently I thought 3 + 1 = 5 😂

Reply:
Fixed in 20bf03a.